Week3/Day2/README.md

```diff
@@ -0,0 +1,16 @@
+## ETL Design Assignment 2.
+
+> ## The file structure of this repo:
+```
+data/             # Folder containing the datasets in different formats.
+  *.json
+  *.csv
+  *.xml 
+docs/           # Folder containing .md files for assignment.      
```

Comment:
Why did two of you had the same typo in this line? @Saphall :thinking: 
https://github.com/sanjeevbanmala/ETL/blame/master/README.md#L10

Reply:
Well, @silwalanish dai told everyone to take my documentation as reference, So mine typo was reflected in others maybe.